main/templates/base.html

```diff
@@ -22,7 +22,7 @@
 
 	<header>
 		<nav class="navbar navbar-default navbar-custom navbar-static-top">
-			<div class="container">
+			<div class="row">
```

Comment:
No reason to use Bootstrap's row class, unless you're putting `col-*` elements inside it.